src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java

```diff
@@ -4,264 +4,304 @@
+                Reporter reporter = subReporter.createSubReporter(STATIC_VAR_COMPENSATOR_KEY, STATIC_VAR_COMPENSATOR_NAME, "id", s.getStaticVarCompensatorId());
+                reporter.report(Report.builder().withKey("staticVarCompensatorNotFound")
+                                                .withDefaultMessage("Static var compensator with id=${id} not found")
+                                                .withValue("id", s.getStaticVarCompensatorId())
+                                                .withSeverity(TypedValue.WARN_SEVERITY).build());
+            } else if (s.getVoltageSetpoint() != null || s.getReactivePowerSetpoint() != null) {
+                modificationsCount++;
+                Reporter reporter = subReporter.createSubReporter(STATIC_VAR_COMPENSATOR_KEY, STATIC_VAR_COMPENSATOR_NAME, "id", s.getStaticVarCompensatorId());
+                reporter.report(Report.builder().withKey("staticVarCompensatorModification")
+                                                .withDefaultMessage("Static var compensator with id=${id} modified :")
+                                                .withValue("id", s.getStaticVarCompensatorId())
+                                                .withSeverity(TypedValue.TRACE_SEVERITY).build());
                 if (s.getVoltageSetpoint() != null) {
-                    double oldTargetV = staticVarCompensator.getVoltageSetpoint();
+                    final double oldTargetV = staticVarCompensator.getVoltageSetpoint();
                     staticVarCompensator.setVoltageSetpoint(s.getVoltageSetpoint());
-                    reporter.report(ModificationUtils.getInstance().buildModificationReportWithIndentation(oldTargetV, s.getVoltageSetpoint(), VOLTAGE_SET_POINT, 1));
+                    reporter.report(ModificationUtils.buildModificationReport(oldTargetV, s.getVoltageSetpoint(), VOLTAGE_SET_POINT, 1, TypedValue.TRACE_SEVERITY));
                 }
                 if (s.getReactivePowerSetpoint() != null) {
-                    double oldTargetQ = staticVarCompensator.getReactivePowerSetpoint();
+                    final double oldTargetQ = staticVarCompensator.getReactivePowerSetpoint();
                     staticVarCompensator.setReactivePowerSetpoint(s.getReactivePowerSetpoint());
-                    reporter.report(ModificationUtils.getInstance().buildModificationReportWithIndentation(oldTargetQ, s.getReactivePowerSetpoint(), REACTIVE_POWER_SET_POINT, 1));
+                    reporter.report(ModificationUtils.buildModificationReport(oldTargetQ, s.getReactivePowerSetpoint(), REACTIVE_POWER_SET_POINT, 1, TypedValue.TRACE_SEVERITY));
                 }
             }
-        });
+        }
+        if (modificationsCount > 0) {
+            subReporter.report(new Report("svcModificationsResume", "${count} static var compensator(s) have been modified.", Map.of(
```

Comment:
```suggestion
            String message = modificationsCount == 1 ? "${count} static var compensator has been modified." : "${count} static var compensators have been modified.";
            subReporter.report(new Report("svcModificationsResume", message, Map.of(
```

Reply:
For clarity purpose, I think using ternary operator like this is not the best